src/components/ContactList/ContactList.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import * as redux from 'react-redux';
+import { shallow } from 'enzyme';
+import ContactList from './ContactList';
+import { contactsMock } from '../../mocks/contactsMock';
+
+describe('ContactList component', () => {
+  const useSelectorMock = jest.spyOn(redux, 'useSelector');
+  const useDispatchMock = jest.spyOn(redux, 'useDispatch');
+
+  const dispatchMock = jest.fn();
+  const deleteContactClickMock = jest.fn();
+  const props = { contactsData: contactsMock };
```

Comment:
delete this line, use with onClick

Reply:
1. `const props` has been updated and used with `onClick`.
2. `const contactList` with `shallow(<ContactList {...props} />)`  has been moved before all tests.